README.md

```diff
@@ -16,3 +16,32 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building these packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the SDK from https://downloads.creatordev.io/pistachio/marduk/OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz.
+2. Extract the SDK and go to the extracted folder.
```

Comment:
Give command with options to extract (e.g. tar -zxvf OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz)